packages/Textarea/src/textarea.scss

```diff
@@ -2,94 +2,145 @@
-  --ids-textarea-char-indicator-right: #{tokens.$space-3};
-  --ids-textarea-char-indicator-color: #{tokens.$grey-400};
-  --ids-textarea-char-indicator-size: #{tokens.$font-size-2};
-  --ids-textarea-char-indicator-line-height: #{tokens.$line-height-xl};
+    /* Default */
+    --ids-textarea-font-family: #{tokens.$primary-font-family};
+    --ids-textarea-font-size: #{tokens.$font-size-4};
+    --ids-textarea-line-height: #{tokens.$line-height-xxxl};
+    --ids-textarea-color: #{tokens.$grey-800};
+    --ids-textarea-border-color: #{tokens.$grey-300};
+    --ids-textarea-border-radius: #{tokens.$border-radius-sm};
+    --ids-textarea-padding: #{tokens.$space-2} #{tokens.$space-3};
+
+    /* Placeholder */
+    --ids-textarea-color-placeholder: #{tokens.$grey-400};
+
+    /* Focus */
+    --ids-textarea-focus-border-color: #{tokens.$electric-blue-500};
+
+    /* Hover */
+    --ids-textarea-hover-border-color: #{tokens.$electric-blue-500};
+    --ids-textarea-hover-background-color: #{tokens.$electric-blue-500};
+
+    /* Error */
+    --ids-textarea-error-border-color: #{tokens.$coral-500};
+
+    /* Disabled */
+    --ids-textarea-disabled-background: #{tokens.$grey-200};
+    --ids-textarea-disabled-color: #{tokens.$grey-500};
+
+    /* Character Indicator */
+    --ids-textarea-char-indicator-bottom: #{tokens.$space-2};
+    --ids-textarea-char-indicator-right: #{tokens.$space-3};
+    --ids-textarea-char-indicator-color: #{tokens.$grey-400};
+    --ids-textarea-char-indicator-size: #{tokens.$font-size-2};
+    --ids-textarea-char-indicator-line-height: #{tokens.$line-height-xl};
+}
+
+[data-brand="workleap"] {
+    /* Default */
+    --ids-textarea-font-family: var(--hop-body-sm-regular-font-family);
+    --ids-textarea-font-size: var(--hop-body-sm-font-size);
+    --ids-textarea-line-height: var(--hop-body-sm-line-height);
+    --ids-textarea-color: var(--hop-neutral-text);
+    --ids-textarea-border-color: var(--hop-neutral-border);
+    --ids-textarea-border-radius: var(--hop-shape-rounded-md);
+    --ids-textarea-padding: var(--hop-space-inset-squish-md);
+
+    /* Placeholder */
+    --ids-textarea-color-placeholder: var(--hop-neutral-text-weakest);
+
+    /* Focus */
+    --ids-textarea-focus-border-color: var(--hop-neutral-border-active);
+    --ids-textarea-focus-box-shadow: 0 0 0 0.12rem var(--hop-neutral-surface), 0 0 0 0.219rem var(--hop-primary-border-focus);
+
+    /* Hover */
+    --ids-textarea-hover-border-color: var(--hop-neutral-border-hover);
+    --ids-textarea-hover-background-color: var(--hop-neutral-surface-hover);
+
+    /* Error */
+    --ids-textarea-error-border-color: var(--hop-danger-border-strong);
+
+    /* Disabled */
+    --ids-textarea-disabled-background: var(--hop-neutral-border-disabled);
+    --ids-textarea-disabled-color: var(--hop-neutral-surface-disabled);
+
+    /* Character Indicator */
+    --ids-textarea-char-indicator-bottom: 0.5rem;
+    --ids-textarea-char-indicator-right: 1rem;
```

Comment:
bottom and right are similar to margin. I would use a token for this.

Reply:
I agree if these emulate a margin I am ok with it.